core/src/main/java/org/arquillian/smart/testing/Logger.java

```diff
@@ -86,6 +88,21 @@ public void warn(String msg, Object... args) {
     }
 
     /**
+     * Will format the given message with the given arguments and prints it on standard output with the prefix:
```

Comment:
I am not sure if we really need this javadoc here. WDYT @MatousJobanek?

Reply:
I'm fine with the javadoc - it describes what it does.
What I can see as a problem is that the log is logged only when `smart.testing.debug=true` is used. This can be confusing as `FINE` and `FINEST` are logged anyway and the visibility can be changed by the JUL log level. IMHO `DEBUG` should be higher than `FINE` or `FINEST`. Our problem is that we mix two or three things together. When we have the `DEBUG` implemented, then I would get rid of JUL and the `FINE` and `FINEST` WDYT?